### DISCUSSION TITLE
Error when using set_brightness

### DISCUSSION THREAD

**Discussion by roboberk** [LightwaveSmartHome/homeassistant-lightwave-smart#13] [Q&A]
Hello,
   Trying to update the dim level of the lights when they are off. Using the "Lightwave Smart custom component: Set Brightness" but every time I run the code I get an error. Please see below:

Error: 'LWRF2Light' object has no attribute '_name'
   
   `service: lightwave_smart.set_brightness
metadata: {}
data:
  brightness: 61
target:
  area_id:
    - kitchen`
  
Does any one know what's going wrong? 

Thank you

**Comment by RMMTSLLP** [comment]
Hi,
Did you ever get "Lightwave Smart custom component: Set Brightness" working I keep getting a error "Failed to perform the action lightwave_smart.set_brightness. extra keys not allowed @ data['brightness']. Got None"
Regards

**Comment by ikb42** [comment]
please see #12
